backend/main.py

```
@@ -2,13 +2,12 @@
 from contextlib import asynccontextmanager
 
 from fastapi import Depends, FastAPI, HTTPException, Query
-from sqlmodel import Field, Session, SQLModel, create_engine, select
+from sqlmodel import Session
 from database import engine, create_db_and_tables, get_session
```

Ruff (0.11.9)

`database.engine` imported but unused

Remove unused import: `database.engine`

(F401)